Adyen/Core/Components/Base/PresentableComponentWrapper.swift

```diff
@@ -52,10 +52,17 @@ public final class PresentableComponentWrapper: PresentableComponent,
 
     /// :nodoc:
     public func didFinalize(with success: Bool) {
-        component.finalizeIfNeeded(with: success)
+        component.finalizeIfNeeded(with: success, completion: {})
```

Comment:
Is there a reason for this empty block, can we send `nil` instead?

Reply:
could be `nil` indeed